share/man/man1/brew.1

```diff
@@ -554,10 +554,6 @@ HOMEBREW_BROWSER
 If set, uses this setting as the browser when opening project homepages, instead of the OS default browser\.
 .
 .TP
-HOMEBREW_BUILD_FROM_SOURCE
```

Comment:
@chdiza It's a strong indication of the quality of your software. No-one likes having issues. If you cannot offer a feature that will work consistently: you should not offer the feature to users but instead let it be installable through some other system (e.g. a tap).